### PR TITLE
node-exporter: do not automount service account token

### PR DESCRIPTION
**Pull request by jonkerj**
#### What this PR does / why we need it:
It disables the automounting of the service account token for node exporter. This exporters does not need access to the Kubernetes API (as far as I can tell). By disabling the automount, potential attackers cannot access the Kubernetes API on behalf/through the pod.

#### Which issue this PR fixes
I have not raised an issue. If it helps or is needed otherwise, I'd be glad to do so.

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by vsliouniaev — DISMISSED** [review]
Thanks, this is a good idea

**Comment by torstenwalter** [comment]
@jonkerj can you resolve the merge conflict?
@vsliouniaev please merge directly after your approval as described in https://github.com/prometheus-community/helm-charts/blob/main/PROCESSES.md#review-process.

**Comment by jonkerj** [comment]
No prob, done and done. Apparently, I "dismissed @vsliouniaev's stale review", sounds more harsh than I meant 😄

**Review by vsliouniaev — APPROVED** [review]

**Pull request merged**
